gui/tasks/forms.py

```diff
@@ -152,6 +161,16 @@ def clean_dayweek(self):
         w = ','.join(w)
         return w
 
+    def clean_transfers(self):
+        value = self.cleaned_data.get('transfers')
+        if value:
+            try:
+                return int(value)
```

Comment:
If all it can be is an integer, why are we using a CharField?

Reply:
I've did it multiple times because django does not have an `IntegerField` that allows empty input.

Reply:
Thats not django, its dojo as far as I know.
Have you tried using a TextInput widget with IntegerField formfield?

Reply:
Yes, `IntegerField` always tries to do `int(value)`

Reply:
Thank you, it worked!